backend/tests/test_sessions_routes.py

```diff
@@ -260,10 +263,16 @@ def test_create_readonly_session_as_user(client, db, user, kubernetes):
     assert session
     assert session.owner_name == user.name
     assert kubernetes.sessions
-    assert (
-        kubernetes.sessions[0]["docker_image"]
-        == "k3d-myregistry.localhost:12345/capella/readonly:5.0.0-latest"
-    )
+    if development_mode:
+        assert (
+            kubernetes.sessions[0]["docker_image"]
+            == "k3d-myregistry.localhost:12345/capella/readonly:5.0.0-latest"
+        )
+    else:
+        assert (
+            kubernetes.sessions[0]["docker_image"]
+            == "ghcr.io/dsd-dbs/capella-dockerimages/capella/readonly:5.0.0-selected-dropins-main"
+        )
```

Comment:
```suggestion
    assert "/capella/readonly:5.0.0" in kubernetes.sessions[0]["docker_image"]
```

This makes the test a little less precise, but still easier to read, since it does not (indirectly) depend on the `DEVELOPMENT_MODE` envvar.